### PR TITLE
CDRW-4347 - add proxy field (and completes CDRW-4344)

### PR DESCRIPTION
**Pull request by charlieGreenyOB**

### CODE REVIEW

**Review by OBPeteS — CHANGES_REQUESTED** [review]
Proxy is a class, please see page 21 of OBL_RC_EAG_Workshop4_26032024.pptx for the schema. Schema will need expanding into data dictionary

File: docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md [hunk 1]
```diff
@@ -142,10 +142,10 @@ For the OBInternationalStandingOrder4 Initiation object:
 | FirstPaymentDateTime |1..1 |OBInternationalStandingOrder4/FirstPaymentDateTime |The date on which the first payment for a Standing Order schedule will be made. |ISODateTime | | |
 | FinalPaymentDateTime |0..1 |OBInternationalStandingOrder4/FinalPaymentDateTime |The date on which the final payment for a Standing Order schedule will be made. |ISODateTime | | |
 | Purpose |0..1 |OBInternationalStandingOrder4/Purpose |Specifies the external purpose code in the format of character string with a maximum length of 4 characters. The list of valid codes is an external code list published separately. External code sets can be downloaded from www.iso20022.org. |OBExternalPurpose1Code1 | | |
-| ExtendedPurpose |0..1 |OBInternationalScheduled3/ExtendedPurpose |Specifies the purpose of an international payment, when there is no corresponding 4 character code available in the ISO20022 list of Purpose Codes. |Max140Text | | |
+| ExtendedPurpose |0..1 |OBInternationalStandingOrder4/ExtendedPurpose |Specifies the purpose of an international payment, when there is no corresponding 4 character code available in the ISO20022 list of Purpose Codes. |Max140Text | | |
```
Comment (OBPeteS) [inline docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md:145]: Extended purpose was meant to be moved to the Risk object as per https://openbanking.atlassian.net/browse/CDRW-4344

File: docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md [hunk 2]
```diff
@@ -142,10 +142,10 @@ For the OBInternationalStandingOrder4 Initiation object:
...
 | ChargeBearer |0..1 |OBInternationalStandingOrder4/ChargeBearer |Specifies which party/parties will bear the charges associated with the processing of the payment transaction. |OBChargeBearerType1Code |BorneByCreditor BorneByDebtor FollowingServiceLevel Shared | |
 | CurrencyOfTransfer |1..1 |OBInternationalStandingOrder4/CurrencyOfTransfer |Specifies the currency of the to be transferred amount, which is different from the currency of the debtor's account. |ActiveOrHistoricCurrencyCode | |^[A-Z]{3,3}$ |
-| DestinationCountryCode |0..1 |OBInternationalScheduled3/DestinationCountryCode |Country in which Credit Account is domiciled. Nation with its own government. |CountryCode | |^[A-Z]{2,2}$ |
+| DestinationCountryCode |0..1 |OBInternationalStandingOrder4/DestinationCountryCode |Country in which Credit Account is domiciled. Nation with its own government. |CountryCode | |^[A-Z]{2,2}$ |
```
Comment (OBPeteS) [inline docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md:148]: Which jira ticket does this change relate to?

**Review by OBPeteS — CHANGES_REQUESTED** [review]
Please see comments on files for requested changes

File: docs/v4.0-draft1/resources-and-data-models/pisp/domestic-payment-consents.md [hunk 3]
```diff
@@ -411,7 +423,7 @@ Content-Type: application/json
     }
   },
   "Links": {
-    "Self": "https://api.alphabank.com/open-banking/v3.1/pisp/domestic-payment-consents/58923"
+    "Self": "https:/api.alphabank.com/open-banking/v3.1/pisp/domestic-payment-consents/58923"
```
[the remark this answers is not in the document]
Comment (charlieGreenyOB) [reply docs/v4.0-draft1/resources-and-data-models/pisp/domestic-payment-consents.md:426]: Aww good spot! I did a replace all for `//` -> `/` as made a mistake in the path in the data dictionary and was not thinking properly

File: docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md (hunk 2 above)
Comment (charlieGreenyOB) [reply docs/v4.0-draft1/resources-and-data-models/pisp/international-standing-order-consents.md:148]: Oh I'm not sure, I just noticed the mistake in the path when going through it, using "the Boy Scout Rule in Coding" - 
Leave it better than you found it

**Review by OBPeteS — APPROVED** [review]

**Pull request merged**